eeg_models/types.py

```diff
@@ -5,7 +5,12 @@
 import torch
 
 
+Any = Any
+Callable = Callable
+Device = torch.device
+Dict = Dict
 Directory = Path
 File = Path
+List = List
 npArray = NewType("nupmy array", np.ndarray)
-Device = torch.device
+Optional = Optional
```

Comment:
remove this changes pls. They are not needed

Reply:
done!